langchain4j-hugging-face/src/main/java/dev/langchain4j/model/huggingface/DefaultHuggingFaceClient.java

```diff
@@ -1,28 +1,28 @@
 package dev.langchain4j.model.huggingface;
 
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+
 import dev.langchain4j.model.huggingface.client.EmbeddingRequest;
 import dev.langchain4j.model.huggingface.client.HuggingFaceClient;
 import dev.langchain4j.model.huggingface.client.TextGenerationRequest;
 import dev.langchain4j.model.huggingface.client.TextGenerationResponse;
+import java.io.IOException;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
 import okhttp3.OkHttpClient;
 import retrofit2.Response;
 import retrofit2.Retrofit;
 import retrofit2.converter.jackson.JacksonConverterFactory;
 
-import java.io.IOException;
-import java.time.Duration;
-import java.util.List;
-
-import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
-
 class DefaultHuggingFaceClient implements HuggingFaceClient {
 
     private static final String BASE_URL = "https://api-inference.huggingface.co/";
 
     private final HuggingFaceApi huggingFaceApi;
     private final String modelId;
 
-    DefaultHuggingFaceClient(String apiKey, String modelId, Duration timeout) {
+    DefaultHuggingFaceClient(Optional<String> baseUrl, String apiKey, String modelId, Duration timeout) {
```

Comment:
We prefer not to use `Optional` for arguments, please use `String baseUrl`

Reply:
I have removed it.